### ISSUE TITLE
bug - 4.78 breaks ui rendering

### ISSUE THREAD

**Issue by zman13** [tinymce/tinymce#4250]
This afternoon we noticed the tinymce textarea were not rendering at all and they were fine on friday. In checking saw that a new version was being pulled from the cdn. Using the cached local version of tinymce fixed the problem which seems to be related to the fix we had in place to deal with dynamic rendering in angular 5. The problem seems to have to do with the code below

```
ngAfterViewInit() {
    tinymce.init({
      mode: 'exact',
      height: 125,
      width: 934,
        theme: 'modern',
        plugins: [
          'advlist autolink lists link image charmap print preview anchor',
          'searchreplace visualblocks code fullscreen',
          'insertdatetime media table contextmenu paste code'
        ],
        toolbar: 'insertfile undo redo | styleselect | bold italic | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent | link',
        elements: "tmce",
        setup: this.tinySetup.bind(this)
    });
    if (tinymce.editors.length > 0) {
      tinymce.execCommand('mceRemoveEditor', true, "tmce");
      tinymce.execCommand('mceAddEditor', true, "tmce");
    }
  }
```
the last 3 lines test to see if there is an editor present already and deletes it and readds. This seems to be causing a problem in the new version.

**Comment by fyrkant** [comment]
Have you tried using our [official Angular wrapper](https://github.com/tinymce/tinymce-angular)? Maybe that works better.

**Comment by spocke** [comment]
This is a duplicate of #4252

**Comment by fyrkant** [comment]
We have now released 4.7.9 that should fix this problem, closing this issue.
